indispensable/src/commonMain/kotlin/at/asitplus/signum/indispensable/io/Encoding.kt

```diff
@@ -128,10 +129,21 @@ fun ByteArray.ensureSize(size: Int): ByteArray = (this.size - size).let { toDrop
 @Suppress("NOTHING_TO_INLINE")
 inline fun ByteArray.ensureSize(size: UInt) = ensureSize(size.toInt())
 
-@Suppress("NOTHING_TO_INLINE")
-inline fun ByteArray.asBuffer()= Buffer().also{it.write(this)}
-
 @OptIn(UnsafeIoApi::class)
 internal fun ByteArray.wrapInUnsafeSource(): Source = Buffer().apply {
     UnsafeBufferOperations.moveToTail(this, this@wrapInUnsafeSource)
 }
+
+@OptIn(UnsafeIoApi::class)
+internal fun wrapInUnsafeSource(vararg bytes: ByteArray) = Buffer().apply {
```

Comment:
add a kdoc describing why this is unsafe

Reply:
reworked as per out offline discussion